### DISCUSSION TITLE
Audio stops shortly after start; maps won't start afterwards.

### DISCUSSION THREAD

**Discussion by Alkenzik** [ppy/osu#20411] [Q&A]
### Type

Game behaviour

### Bug description

After about 4 minutes after launching, all audio stops. When a beatmap starts, the mapping and time indicator do not advance.
Error occurred on EndeavourOS, an Arch based distribution, running the i3 window manager. 

![2022-09-22-22:38:06-screenshot](https://user-images.githubusercontent.com/105467710/191893011-c4aca2c8-a22c-4230-a24a-6f261af3dabb.png)

### Version

2022.910.0

### Logs

[database.log](https://github.com/ppy/osu/files/9630715/database.log)
[network.log](https://github.com/ppy/osu/files/9630716/network.log)
[performance.log](https://github.com/ppy/osu/files/9630717/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9630718/runtime.log)

**Comment by peppy** [comment]
Try reading some of the [existing](https://github.com/ppy/osu/search?q=linux+audio&type=discussions) discussions regarding linux audio, but in general you're probably going to have to figure things out yourself.

**Reply by Alkenzik** [reply]
I see. Thanks for responding.
